### PR TITLE
Proper entity annotations, backend application can run

### PR DESCRIPTION
**Pull request by wcollins37**

### CODE REVIEW

File: findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Calendar.java [hunk 1]
```diff
@@ -1,6 +1,10 @@
+/*
 package com.tp.wrc.findmyparadise.models;
 
+<<<<<<< HEAD
```
Comment (zmiles17) [inline findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Calendar.java:4]: This looks like some left over from a merge conflict?

File: findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Calendar.java [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+=======
```
Comment (YesratRahman) [inline findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Calendar.java:5]: Fix the conflicts

**Review by kpugh97 — CHANGES_REQUESTED** [review]
left over merge conflicts

**Review by zmiles17 — COMMENTED** [review]
I see a lot of commented out code and I'm assuming it was to get the program to compile, but I think in general it's not a great practice.

**Pull request closed without merging**